.travis.yml

```diff
@@ -10,14 +10,11 @@ branches:
 language: php
 
 php:
-  - '5.6'
-  - '7.0'
```

Comment:
@ElectricMaxxx i found that you droped support for PHP5.6 here https://github.com/symfony-cmf/testing/pull/146 and for PHP7.0 here https://github.com/symfony-cmf/testing/pull/155 in `symfony-cmf/testing`. But this bundle still supports that versions!

So, i created a PR to test how `symfony-cmf/testing` works with PHP 5.6 and PHP 7.0 https://github.com/symfony-cmf/testing/pull/181. I solved a "composer and out or memory" issue, and travis is green these. See for yourself https://travis-ci.org/symfony-cmf/testing/builds/396024428